Assets/Scripts/GameData/Abilities/IAbility.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SwordAndBored.GameData.Abilities
+{
+    interface IAbility : IDescriptable
+    {
+        /// <summary>
+        /// The name of the image used to represent this ability
+        /// </summary>
+        string ImageName { get; }
+    }
```

Comment:
Can we include ImageName in a different abstraction? Because I will probably need to represent buildings with images in the base UI

Reply:
resolved in a8cd9b3